src/components/gataca-qrdisplay/readme.md

```diff
@@ -0,0 +1,63 @@
+# Gataca-QR Display
+
+This component built using stencyl allows an easy integration to display a QR with the gataca styles
+
+Continuing with that example, you could integrate in any HTML using the following code
+
+````html
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0">
+  <title>Gataca QR Display Component</title>
+  <script type="module" src="/build/gataca-qrdisplay.esm.js"></script>
+  <script nomodule src="/build/gataca-qrdisplay.js"></script>
+</head>
+
+<body>
+  <div class="your-container-styles">
+
+    <gataca-qrdisplay id="gataca-qr" qr-data="DATA TO DISPLAY">
+      </gataca-qr>
+
+  </div>
+</body>
+
+</html>
+````
+
+<!-- Auto Generated Below -->
+
+
+## Properties
+
+| Property   | Attribute   | Description                                                                                              | Type      | Default     |
+| ---------- | ----------- | -------------------------------------------------------------------------------------------------------- | --------- | ----------- |
+| `logoSize` | `logo-size` | _[Optional]_ Size of the logo to display. 0 means no logo will be displayed. Default is the GATACA logo. | `number`  | `0`         |
+| `logoSrc`  | `logo-src`  | _[Optional]_ Logo to display, just if the logo size is greater than 0. No logo is the GATACA logo.       | `string`  | `undefined` |
```

Comment:
As I understood we decided to not change the branding of the component

Reply:
logoSize 1,2,3 what means?

Reply:
It's a number between 0 and 1, being a percentage of the size. Recomended is between 0.3 and 0.4, we use 0.33

Reply:
I think, it must be included into the Readme